src/components/ProfileHeader.jsx

```diff
@@ -0,0 +1,178 @@
+import User1 from "../assets/user/user1.svg";
+import {
+  DotsThree,
+  CaretLeft,
+  CaretDown,
+  Envelope,
+} from "@phosphor-icons/react/dist/ssr";
+import useMediaQuery from "../hooks/useMediaQuery";
+
+const ProfileHeader = ({ user }) => {
+  const isAboveMediumScreens = useMediaQuery("(min-width: 730px)");
```

Comment:
Mobile version should have been added in a different PR. There was no need to further complicate. Please separate.

Reply:
I meant that responsiveness should be added in a separate PR all together.